keps/sig-release/20190118-creating-packages.md

```diff
@@ -0,0 +1,417 @@
+---
+kep-number: $i
+title: Packaging of kubernetes artefacts
```

Comment:
i am so used to seeing "artefact" written as "artifact" that i forgot both are correct. one is british and the other american english. :)

Reply:
OK. I will globally change to "artifact". I am neither a native Brit nor a native US American -- so I don't care ;)